ui/css/logviewer.css

```diff
@@ -135,16 +113,11 @@ body {
 }
 
 .run-data .break-word{
-    -ms-word-break: break-all;
```

Comment:
This removal was based on the [word-break](https://developer.mozilla.org/en-US/docs/Web/CSS/word-break) MDN compatibility note stating it shouldn't be used (see 'Browser Compatibility' [1] ). I don't have ready IE access so can't verify that is the case on my end.